cmd/bootstrap/cmd/qc.go

```diff
@@ -20,7 +20,13 @@ func constructRootQC(block *flow.Block, votes []*model.Vote, allNodes, internalN
 		log.Fatal().Err(err).Msg("failed to generate QC participant data")
 	}
 
-	qc, err := run.GenerateRootQC(block, votes, participantData, identities)
+	qc, invalidVotesErr, err := run.GenerateRootQC(block, votes, participantData, identities)
+	if len(invalidVotesErr) > 0 {
+		for _, err := range invalidVotesErr {
+			log.Warn().Err(err).Msg("invalid vote")
```

Comment:
Will print like this even if a QC is built.

<img width="1660" alt="image" src="https://user-images.githubusercontent.com/811374/213558416-96712367-400c-4f78-9941-9c4da6c4c09b.png">